From bf8b4362cef24a6f747e4329305a4939c8f585d1 Mon Sep 17 00:00:00 2001
From: David King <amigadave@amigadave.com>
Date: Thu, 2 Jul 2015 13:49:22 +0100
Subject: [PATCH] Fix systemd service file

udisks-daemon is installed to $(libexecdir), not $(prefix)/lib/udisks.

https://bugzilla.redhat.com/show_bug.cgi?id=1238664
https://bugs.freedesktop.org/show_bug.cgi?id=91191

[This patch has been backported from upstream:
https://cgit.freedesktop.org/udisks/commit/?h=udisks1&id=bf8b4362cef24a6f747e4329305a4939c8f585d1]
Signed-off-by: Pieterjan Camerlynck <pieterjan.camerlynck@gmail.com>
---
 data/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/Makefile.am b/data/Makefile.am
index 411ea0f..08af5f4 100644
--- a/data/Makefile.am
+++ b/data/Makefile.am
@@ -31,7 +31,7 @@ systemdservicedir       = $(systemdsystemunitdir)
 systemdservice_DATA     = $(systemdservice_in_files:.service.in=.service)
 
 $(systemdservice_DATA): $(systemdservice_in_files) Makefile
-	@sed -e "s|\@libexecdir\@|$(prefix)/lib/udisks|" $< > $@
+	@sed -e "s|\@libexecdir\@|$(libexecdir)|" $< > $@
 endif
 
 udevrulesdir = $(slashlibdir)/udev/rules.d
-- 
2.7.4

